docs/units/standard_units.rst

```diff
@@ -162,7 +162,26 @@ For example::
    Unit(dimensionless with a scale of 1000.0)
    >>> (u.km / u.m).decompose() == u.dimensionless_unscaled
    False
-
+   
+To create a dimensionless quantity with a specific scaling factor, 
+multiply or divide a value by ``Unit()`` with the scaling 
```

Comment:
Here again I would just describe multiplication, and write `Unit(scale_factor)`. I'd also give the thing we do *not* recommend in the text rather than as an actual example, maybe as `mega_factor = 20`.

Reply:
Also, might it make sense to tell that `u.percent` is implemented exactly like this?